beacon-chain/core/epoch/precompute/reward_penalty_test.go

```diff
@@ -55,15 +55,15 @@ func TestProcessRewardsAndPenaltiesPrecompute(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	// Indices that voted
-	wanted := uint64(32000001776)
+	// Indices that voted everything except for head, lost a bit money
+	wanted := uint64(31999995452)
```

Comment:
can we calculate this number easily?

Reply:
It's pretty tedious, i think what we have is sufficient enough for unit test, the rest is covered as part of spec test

Reply:
sg. thanks